zulipterminal/ui_tools/boxes.py

```diff
@@ -710,6 +710,14 @@ def autocomplete_emojis(
 
         return emoji_typeahead, emojis
 
+    def insert_uri(self, file_name: str, uri: str) -> None:
+        edit_widget = self.contents[self.FOCUS_CONTAINER_MESSAGE][
+            self.FOCUS_MESSAGE_BOX_BODY
+        ]
+        edit_widget.edit_text += f"[{file_name}](/{str(uri)})"
```

Comment:
Upon sending the message  and opening the file, the URL wasn't able to fetch the file. This may be because of the extra `/` before the `uri`, because the URI returned from the server already has a `/` at the start.

Also, the function already has the type of `uri` as `str`, so there might be no need to typecast it to string.